OxfordBodleian/Juel-Jensen/BDLaethf30.xml

```diff
@@ -170,59 +186,99 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                  </list>
                              </handNote>
                          </handDesc>
-                     
-                        
+
+
                         <additions>
                             <list>
                                 <item xml:id="e1">
                                     <locus target="#ir"/>
                                     <desc>A white sticker is glued to <locus target="#ir"/>. The name of <persName ref="PRS5782JuelJen"/> is printed
                                         on it
-                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signatures "MS. Aeth. f. 30" and 
+                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signatures "MS. Aeth. f. 30" and
                                         "MS Et 5" are written in pencil.
                                         An additional sticker, with the signature "Ms 5" on it, is glued below it.</desc>
                                 </item>
-                                
+
                                 <item xml:id="e2">
                                     <locus target="#ir"/>
                                     <desc>Recent note written in pencil and barely legible, containing minimal descriptive information on the manuscript.</desc>
-                                    <q xml:lang="en">Ethiopic <gap reason="illegible"/> Psalter followed by usual odes 
+                                    <q xml:lang="en">Ethiopic <gap reason="illegible"/> Psalter followed by usual odes
                                         <gap reason="illegible"/>
                                     late C 19 early C 20</q>
                                 </item>
                             </list>
                         </additions>
-                        
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
-                        leaf string marker f.  39, 101, 104
+
+                        <bindingDesc>
+                            <binding xml:id="binding" contemporary="true Ethiopian">
+                              <p>
+                                The manuscript was repaired several times, as testify the various threads visible on the spine and holes drilled through the boards.
+                                However, the materials and the techniques used are compatible with those of the Ethiopian binding tradition.
+                                The boards and the cover are possibly contemporary with most of the manuscript content.</p>
```

Comment:
I would say "possibly contemporary with the main texts" (which recalls the hierarchy adopted in the textual part)